geo/walk.py

```diff
@@ -0,0 +1,46 @@
+import networkx as nx
+import osmnx as ox
+from geopandas import GeoDataFrame
+from networkx import MultiDiGraph
+from shapely.geometry.point import Point
+from typing import Tuple
+'''Library to support geographical computations .'''
```

Comment:
Comment line goes at the top, and there should be a blank line before the imports start.